src/xhr/lib/errorReporter.js

```diff
@@ -0,0 +1,34 @@
+import XhrError from './XhrError';
+// TODO: Add in Sentry
+export default {
+    captureException: (errorOrResponse) => {
+        if (process.env.NODE_ENV === 'production')
+            return;
+        let name = 'XhrError';
+        let message;
+        let response;
+        let stack = [];
+        if (errorOrResponse instanceof XhrError) {
```

Comment:
This is a slight departure from where we left FYSTS upon merging https://github.com/getethos/FYSTS/pull/7 ... it turned out checking for `typeof errorOrResponse.ok` against `errorOrResponse` when it's an instance of `XhrError` was angering Typescript in this environment since `XhrError` has no `.ok` property. Changing it to this made more sense to me anyway, and removed the compilation error as well so I went with it.